test/all/index.js

```diff
@@ -18,6 +18,11 @@ const fileContent = new Buffer(
   'IM A LUMBERJACK AND IM OK, I SLEEP ALL NIGHT AND I WORK ALL DAY'
 );
 
+const file2 = 'foobaz.txt';
```

Comment:
Do we need a second file for this?

As the code stands, we def do. If we move the `upload` test below `delete` as proposed below, we can re-use the original file.